Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -79,6 +80,94 @@
+    globally_prevalence: int    # same as 'global_prevalence', but as integer
+    global_first_observed: str
+    global_last_observed: str
+    top_file_names: list[str]
+
+    @classmethod
+    def from_raw_response(cls, raw_response: dict):
+        """Creates an instance from the file stats API raw response body (ignores extra fields, if any).
+
+        Args:
+            raw_response (dict): File stats API response
+
+        Returns:
+            FileStatisticsAPIParser
+        """
+        dataclass_field_names = {field.name for field in dataclasses.fields(cls)}
+        snake_case_response = snakify(raw_response)
+        return cls(**{key: value for key, value in snake_case_response.items() if key in dataclass_field_names})
+
+    def to_context_output(self) -> dict:
+        """Generates context output from an instance of FileStatisticsAPIParser.
+
+        Returns:
+            dict: context output
+        """
+        return {
+            'Sha1': self.sha1,
+            'Statistics': assign_params(
+                **{camelize_string(key): value for key, value in dataclasses.asdict(self).items() if key != 'sha1'}
+            )
+        }
+
+    def to_human_readable(self, file_hash: str) -> str:
+        """Generates a human readable table from an instance of FileStatisticsAPIParser.
+
+        Args:
+            file_hash (str): The hash of the file
+
+        Returns:
+            str: human readable markdown table
+        """
+        table_data = {self.format_for_table(key): value for key, value in dataclasses.asdict(self).items() if key != 'sha1'}
+        return tableToMarkdown(f'Statistics on {file_hash} file:', table_data, removeNull=True)
+
+    def to_file_indicator(self, file_hash: str) -> Common.File:
+        """Generates a File indicator object from an instance of FileStatisticsAPIParser.
+
+        Args:
+            file_hash (str): The hash of the file
+
+        Returns:
+            Common.File
+        """
+        return Common.File(
+            dbot_score=Common.DBotScore(file_hash, DBotScoreType.FILE, INTEGRATION_NAME, Common.DBotScore.NONE),
+            sha1=self.sha1,
+            organization_prevalence=self.organization_prevalence,
+            global_prevalence=self.globally_prevalence,
+            organization_first_seen=self.org_first_seen,
+            organization_last_seen=self.org_last_seen,
+            first_seen_by_source=self.global_first_observed,
+            last_seen_by_source=self.global_last_observed,
+        )
+
+    @staticmethod
+    def format_for_table(field_name: str) -> str:
+        """Replaces certain words and formats fields from 'snake_case' to 'Space Case'.
+
+        Args:
+            field_name (str): Name of field in snake_case.
+
+        Returns:
+            str: Formatted in Space Case with replacements.
+        """
+        replacements = [('globally_', 'global_'), ('org_', 'organization_')]
+        for old_value, new_value in replacements:
```

Comment:
```suggestion
        replacements = {'globally_': 'global_', 'org_': 'organization_'}
        for old_value, new_value in replacements.items():
```

Reply:
Fixed in commit d19600a189c37cd368f75dcb76ca7b2f84b28887